example_tally/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e 
+
+# export LIBBABELTRACE2_INIT_LOG_LEVEL=DEBUG
+# export BABELTRACE_CLI_LOG_LEVEL=DEBUG
+
+cd SINK.xprof
+rm -f component.h dispatch.{h,c,o} params.{h,c,o} xprof.{c,o,so} my_callbacks.o my_demangle.o
+cd -
+
+ruby ../main.rb -u 1.stream_classes.yaml -t SINK -p metababel2
+make -C SINK.xprof
+
+babeltrace2 \
+	--plugin-path=/soft/debuggers/thapi/0.11.2/lib:./SINK.xprof \
+	--component=filter.ompinterval.interval \
+	--component=filter.zeinterval.interval \
+	--component=filter.cudainterval.interval \
+	--component=filter.clinterval.interval \
+	--component=sink.metababel2.xprof \
+	/home/avivasmeza/aurelio
```

Comment:
This will not work : ) Please create a new source, with adequate messages to tests the tally.